### DISCUSSION TITLE
Hello, world! 🤖

### DISCUSSION THREAD

**Discussion by exvuma** [community/community#10938] [Code Security]
The Dependabot category is a place for us to discuss feedback, feature requests, common pitfalls, or just to say hello or “thanks” to your friends at GitHub working on Dependabot. You can also upvote existing requests to get our attention with a simple 👍🏼 👎🏼 . 

## What is Dependabot? 

Dependabot is the friendly update bot that lets you know when you have a security vulnerability with [Dependabot alerts](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/about-alerts-for-vulnerable-dependencies), and helps you update that dependency with [Dependabot security update PRs](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/about-dependabot-security-updates). Dependabot also keeps your dependencies up to date more generally with [Dependabot version updates](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/about-dependabot-version-updates).

## What about dependabot-core?

GitHub maintains an open-source project called [dependabot-core](https://github.com/dependabot/dependabot-core), which is the core logic of Dependabot PR updates. If you have questions, feedback, or feature requests for dependabot-core, this discussion is still the right place!

## Helpful Links

- [Dependabot Security and Version Auto-update Documentation](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically) 
- [Dependabot Alerts Documentation](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/about-alerts-for-vulnerable-dependencies)
- Check out our feedback [ README](https://github.com/github/feedback#github-public-feedback-discussions) and [code of conduct](https://github.com/github/feedback/blob/main/CODE_OF_CONDUCT.md) to learn more about how GitHub engages with feedback.

## Still need help?

- For bug reports or other user-specific issues with Dependabot Alerts or UI, head to [support.github.com](https://support.github.com/).
-  For issues like bugs and defects in the logic unique to your language (e.g. parsing manifest files), [file an issue](https://github.com/dependabot/dependabot-core/issues) instead of opening up a discussion thread. 

Note: we aren’t accepting new ecosystems at this time but are working on a long-term plan to make extending to a variety of ecosystems scalable.

**Comment by bandirevanth** [comment]


**Reply by Jasonvl1** [reply]
🙏🏽🙏🏽

**Comment by Rwagner548** [comment]
Hello and thanks for the information.

**Reply by shadow145866** [reply]
> Hello and thanks for the information.

**Comment by P7-33** [comment]
@Jasonvl1

**Comment by toandd7189** [comment]
thanks for infomation

**Comment by shadow145866** [comment]
Hello world

**Reply by Plusheaven** [reply]
Hello back 🔙
This is KryptoKevin speaking to the masses
Crypto is here to stay now regulations or not 🚫

**Comment by Donabbaskid** [comment]
> The Dependabot category is a place for us to discuss feedback, feature requests, common pitfalls, or just to say hello or “thanks” to your friends at GitHub working on Dependabot. You can also upvote existing requests to get our attention with a simple 👍🏼 👎🏼 .
> 
> ## What is Dependabot?
> Dependabot is the friendly update bot that lets you know when you have a security vulnerability with [Dependabot alerts](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/about-alerts-for-vulnerable-dependencies), and helps you update that dependency with [Dependabot security update PRs](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/about-dependabot-security-updates). Dependabot also keeps your dependencies up to date more generally with [Dependabot version updates](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/about-dependabot-version-updates).
> 
> ## What about dependabot-core?
> GitHub maintains an open-source project called [dependabot-core](https://github.com/dependabot/dependabot-core), which is the core logic of Dependabot PR updates. If you have questions, feedback, or feature requests for dependabot-core, this discussion is still the right place!
> 
> ## Helpful Links
> * [Dependabot Security and Version Auto-update Documentation](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically)
> * [Dependabot Alerts Documentation](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/about-alerts-for-vulnerable-dependencies)
> * Check out our feedback [ README](https://github.com/github/feedback#github-public-feedback-discussions) and [code of conduct](https://github.com/github/feedback/blob/main/CODE_OF_CONDUCT.md) to learn more about how GitHub engages with feedback.
> 
> ## Still need help?
> * For bug reports or other user-specific issues with Dependabot Alerts or UI, head to [support.github.com](https://support.github.com/).
> * For issues like bugs and defects in the logic unique to your language (e.g. parsing manifest files), [file an issue](https://github.com/dependabot/dependabot-core/issues) instead of opening up a discussion thread.
> 
> Note: we aren’t accepting new ecosystems at this time but are working on a long-term plan to make extending to a variety of ecosystems scalable.

**Comment by Donabbaskid** [comment]
Hassan MoHAMMAD

**Comment by 123alexsandr** [comment]
Попробуем

**Comment by ghost** [comment]
你好
